plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -1220,6 +1220,94 @@ public static Object[][] sortedTableWithSortTransform()
         };
     }
 
+    protected abstract boolean isFileSorted(String path, String sortColumnName);
+
+    @Test
+    public void testSortedNationTable()
+    {
+        Session withSmallRowGroups = withSmallRowGroups(getSession());
+        String tableName = "test_sorted_nation_table_" + randomNameSuffix();
+        assertUpdate(withSmallRowGroups, "CREATE TABLE " + tableName + " WITH (sorted_by = ARRAY['comment']) AS SELECT * FROM nation", 25);
+        for (Object filePath : computeActual("SELECT file_path from \"" + tableName + "$files\"").getOnlyColumnAsSet()) {
+            assertTrue(isFileSorted((String) filePath, "comment"));
```

Comment:
do we have a test sorting desc
do we have a test sorting by two columns
and covering values containing nulls?

Reply:
All are covered by the SparkCompatibilityTests, but I can duplicate them here too if you'd like.